components/VercelChat/chat.tsx

```
@@ -20,7 +20,7 @@ import { UIMessage } from "ai";
 import { useDropzone } from "@/hooks/useDropzone";
 import FileDragOverlay from "./FileDragOverlay";
 import { Loader } from "lucide-react";
-import { memo, useCallback } from "react";
+import { memo, useCallback, useEffect } from "react";
```

ESLint

[error] 'useEffect' is defined but never used.

(@typescript-eslint/no-unused-vars)